store/models.py

```diff
@@ -30,3 +31,11 @@ def __str__(self):
         else:
             return f'{self.book.title} - Available'
 
+
+class RatingSystem(models.Model):
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    rated_by = models.ForeignKey(User, on_delete=models.CASCADE) 
+    rating = models.SmallIntegerField(default=None)
```

Comment:
You can also use `MinValueValidator`,  `MaxValueValidator` for this as validators at database level.